.github/workflows/on-config-pullrequest-dockerfile.yml

```diff
@@ -57,7 +60,7 @@ jobs:
           cd $folder
           dockerfile=$(echo ${{matrix.dockerfile.path}} | awk -F"/" '{print $NF}')
 
-          docker build -f $dockerfile -t ${{ matrix.dockerfile.name }} .
+          docker buildx build -f $dockerfile -t ${{ matrix.dockerfile.name }} . --build-context parentdir=../
```

Comment:
`docker build` works just fine, but now it is an alias to `docker buildx build`. I prefer being more clear about what we're doing by using `buildx`.